client/src/Movies/MovieCard.js

```diff
@@ -1,7 +1,25 @@
 import React from 'react';
 
 const MovieCard = props => {
-  return;
-};
+  const { title, director, metascore, stars } = movie;
```

Comment:
Good! So to review, that's the same as:
`const title = movie.title;`
`const director = movie.director`
etc...